docs/layouts/shortcodes/cli-installation-check.md

```diff
@@ -0,0 +1,17 @@
+Run the CLI in your terminal with the command:
+
+```bash
+platform
+```
+
+On the first run, you need to log in using a browser.
```

Comment:
:pick: I think the original phrasing (without `On the first run`) was better. These instructions aren't about running it multiple times. It's a step-by-step guide and was describing what happens at this step. Rather than a general statement about what happens every time.